### PR TITLE
Bundle localizations

### PR DESCRIPTION
**Pull request by wiredearp**
@zdlm @sampi @kaumac 

Concat localization files into the main `ts.js` to release only a single JS file going forward (see https://github.com/Tradeshift/tradeshift-ui/issues/159). This means that we can consistently spiritualize on `DOMContentLoaded` because we don't need to wait around for another script to download; and the framework also becomes easier to package in an NPM module where it (potentially) becomes required/bundled instead of outright loaded from the internet; and it becomes easier for devs to download and modify `ts.js` locally in case they encounter a critical bug that needs an immediate hotfix; and it is also just simpler and greater 🍺 

Also fixes https://github.com/Tradeshift/tradeshift-ui/issues/472 - Time component labels are not translated

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -696,4 +675,51 @@ module.exports = function(grunt) {
...
+					.map(line => '\t\t' + line)
+					.join('\n') +
+				'break;'
+			);
+		};
+	}
+
+	/**
+	 * Moment.js has been copy-pasted from the internet with *all* the possible 
+	 * language code and that is considerably more than we need, so we'll remove
```
Comment (sampi) [inline Gruntfile.js]: small typo in `codes`

File: Gruntfile.js [hunk 2]
```diff
@@ -696,4 +675,51 @@ module.exports = function(grunt) {
...
+		};
+	}
+
+	/**
+	 * Moment.js has been copy-pasted from the internet with *all* the possible 
+	 * language code and that is considerably more than we need, so we'll remove 
+	 * the unneeded translations based on the list of supported language codes.
+	 * These translations fortunately follow a uniform pattern so that we can 
+	 * simply include or reject them based on advanced analysis of JS comments.
+	 * Note: The language code `nb` is equivalent to our wrongly name `no` code.
```
Comment (sampi) [inline Gruntfile.js]: small typo in `named`

File: Gruntfile.js [hunk 3]
```diff
@@ -696,4 +675,51 @@ module.exports = function(grunt) {
...
+	 * @returns {Function}
+	 */
+	function getmomentprocessor(codes) {
+		return src => {
+			const split = '//! moment.js locale configuration';
+			const short = [...codes].map(s => s.split('-')[0]);
+			const first = s => (s = s.trim()).slice(0, s.indexOf('\n'));
+			const trail = s => s.split('\n});')[1] || '';
+			const match = s => ((s = s.match(/\[(.*)\]/)) ? s[1] : null);
+			const patch = s => s === 'nb';
```
Comment (sampi) [inline Gruntfile.js]: shouldn't we just have a file called `nb` and another `no` so we support both?

File: docs/src/xhtml/components/times/langs.xhtml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+		<main data-ts="Main">
+			<article>
+				<h1>Time</h1>
+				<section class="desc">
+					<p>TODO: This should go into a screenshot for the test suite!</p>
+				</section>
+				<section data-ts="Spirit">
+					<script type="text/edbml">
+						<table>
+							('da de-CH de en-GB en-US en es ' +
```
Comment (sampi) [inline docs/src/xhtml/components/times/langs.xhtml]: cool thing could be to move the locale-detection code in a shared file between ui and docs, so this page could just use that during generation.

that way whenever we add new languages they will magically show up. I guess this all should belong on a separate page showing what languages we support for dates and such.

File: src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/dates/ts.ui.TimeSpirit.js [hunk 5]
```diff
@@ -1,21 +1,43 @@
...
-ts.ui.TimeSpirit = (function using(chained) {
+ts.ui.TimeSpirit = (function using(moment, chained) {
 	var mytick = 'mytick';
 	var time = 1000;
 	var issame = false;
 	var firsttime = null;
 
+	/**
+	 * There's two problems. First, our language code `no` for Nynorsk is not 
+	 * correctly declared in V4 as `nn-NO` according to the ISO standard and
```
Comment (sampi) [inline src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/dates/ts.ui.TimeSpirit.js]: small typo in `nb-NO`

File: src/runtime/js/ts.ui/lang/ts-lang-da.js [hunk 6]
```diff
@@ -1,36 +1,34 @@
...
-			'Okt',
-			'Nov',
-			'Dec'
-		],
-		dayNames: ['Søndag', 'Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag'],
-		dayNamesShort: ['Søn', 'Man', 'Tir', 'Ons', 'Tor', 'Fre', 'Lør'],
-		dayNamesMin: ['Sø', 'Ma', 'Ti', 'On', 'To', 'Fr', 'Lø']
-	});
-}
+ts.ui.DatePicker.localize({
```
Comment (sampi) [inline src/runtime/js/ts.ui/lang/ts-lang-da.js]: noice

File: Gruntfile.js (hunk 3 above)
Comment (wiredearp) [reply Gruntfile.js]: 'no' doesn't exist, we made that up, that string is however emitted in V4 vis some obscure Grails config, so users will see the `nb` file if we create that; and also nobody will maintain it, or in best case maintain the wrong version 🙈

File: src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/dates/ts.ui.TimeSpirit.js (hunk 5 above)
Comment (wiredearp) [reply src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/dates/ts.ui.TimeSpirit.js]: No, `nn-NO` is the certified ISO language code that we wrongly tried to target with `no`.

File: docs/src/xhtml/components/times/langs.xhtml (hunk 4 above)
Comment (wiredearp) [reply docs/src/xhtml/components/times/langs.xhtml]: Agreed, but let's call that a feature creep for now.

File: Gruntfile.js (hunk 3 above)
Comment (wiredearp) [reply Gruntfile.js]: What happens here is that we keep the `nb` locale in Moment and then we just-in-time translate `no` to `nb` during runtime.

**Pull request merged**